addons/html_builder/static/src/website_builder_action.js

```diff
@@ -176,6 +178,7 @@ export class WebsiteBuilder extends Component {
 
     toggleMobile() {
         this.state.isMobile = !this.state.isMobile;
+        this.websitePreviewRef.el.classList.toggle("o_is_mobile");
```

Comment:
nitpick: There should not be any problem but I think I would do `this.websitePreviewRef.el.classList.toggle("o_is_mobile", this.state.isMobile);` to be sure there is no inconsistency between `this.state.isMobile` and the presence of the `o_is_mobile` class